src/layer/S1GRDAWSEULayer.ts

```diff
@@ -122,6 +137,7 @@ export class S1GRDAWSEULayer extends AbstractSentinelHubV3Layer {
       this.orthorectify = layerParams['orthorectify'];
       this.demInstanceType = layerParams['demInstance'];
       this.orbitDirection = layerParams['orbitDirection'] ? layerParams['orbitDirection'] : null;
+      this.speckleFilter = layerParams['speckleFilter'];
```

Comment:
This - and it should be fixed for other params as well - shouldn't override the existing values, if they are set.

This is most likely the reason why it currently doesn't work in EOB.